src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,56 @@
 
 class Animal {
   // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  die() {
+    Animal.alive = Animal.alive.filter(animal => animal !== this);
+  }
+
+  afterBite(decreaseInHealth) {
```

Comment:
```suggestion
  reduceHeatlh(decreaseInHealth) {
```
I think that's a little better name for that method